lib/jekyll.rb

```diff
@@ -32,6 +32,9 @@ def require_all(path)
 require "colorator"
 
 SafeYAML::OPTIONS[:suppress_warnings] = true
+ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
```

Comment:
There will be windows support. But it'll be a much simpler method we use as
we don't need such wide support. This was originally built for a Docker
image so everything here reflects that.

On Sat, Sep 10, 2016, 5:59 AM Saleem Rashid <notifications@github.com>
wrote:

> In lib/jekyll.rb
> <https://github.com/jekyll/jekyll/pull/5344#discussion_r78275632>:
>
> > @@ -32,6 +32,9 @@ def require_all(path)
> >  require "colorator"
> >
> >  SafeYAML::OPTIONS[:suppress_warnings] = true
> > +ENV["PARALLEL_UNITS"] ||= Pathutil.new("/proc/cpuinfo").read
>
> OK, I was just wondering about Windows support On 10 Sep 2016 11:58,
> Jordon Bedwell <notifications@github.com> wrote: In lib/jekyll.rb<#5344
> (comment)
> <https://github.com/jekyll/jekyll/pull/5344#discussion_r78275614>>:
> @@ -32,6 +32,9 @@ def require_all(path) require "colorator"
> SafeYAML::OPTIONS[:suppress_warnings] = true +ENV["PARALLEL_UNITS"] ||=
> Pathutil.new("/proc/cpuinfo").read
> I know what it does. I audited the code before I even used the library. It
> existence doesn't mean I have to use it or even accept it. I made my choice
> and it was to build my own as noted. … On Sat, Sep 10, 2016, 5:52 AM Saleem
> Rashid ***@***.***> wrote: In lib/jekyll.rb <#5344
> <https://github.com/jekyll/jekyll/pull/5344> (comment)<#5344 (comment)
> <https://github.com/jekyll/jekyll/pull/5344#discussion_r78275538>>>: > @@
> -32,6 +32,9 @@ def require_all(path) > require "colorator" > >
> SafeYAML::OPTIONS[:suppress_warnings] = true > +ENV["PARALLEL_UNITS"] ||=
> Pathutil.new("/proc/cpuinfo").read Parallel.processor_count does the same
> thing, it's cross-platform though. You're using parallel anyway, so it
> makes sense to use it?
> https://github.com/grosser/parallel/blob/master/lib/parallel/processor_count.rb
> — You are receiving this because you were mentioned. Reply to this email
> directly, view it on GitHub <
> https://github.com/jekyll/jekyll/pull/5344/files/f8e64e20e47f8bbb29c7b202747a1875a058ada8#r78275538>,
> or mute the thread <
> https://github.com/notifications/unsubscribe-auth/AAGFs-GPpZIObcjmXL95MzpTqY6wUCjXks5qoovPgaJpZM4J5fpq>
> . — You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub<
> https://github.com/jekyll/jekyll/pull/5344/files/f8e64e20e47f8bbb29c7b202747a1875a058ada8#r78275614>,
> or mute the thread<
> https://github.com/notifications/unsubscribe-auth/AFi_qUjceezkk2KFelj2n-otZi-nURqJks5qoo01gaJpZM4J5fpq
> >.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5344/files/f8e64e20e47f8bbb29c7b202747a1875a058ada8#r78275632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGFs9OB0Mr26bw_tCxJhEXy2TP2Vbi5ks5qoo2AgaJpZM4J5fpq>
> .
>